installer_scripts/install-voyager-airgapped.sh

```diff
@@ -685,6 +696,19 @@ ubuntu_main() {
         # Call the install function with module details
         install_perl_module "$module_name" "$requirement_type" "$required_version" "$package"
     done
+    sudo dpkg -i ./libdbd-mysql-perl*.deb 1>&2
```

Comment:
for rhel, you also install something called mariadb-connector. Not required in ubuntu?

Reply:
Nope not required in this case. We tested it out.